mergify_engine/tests/functional/actions/test_edit.py

```diff
@@ -0,0 +1,54 @@
+# -*- encoding: utf-8 -*-
+#
+#  Copyright © 2021 Mergify SAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import yaml
+
+from mergify_engine.tests.functional import base
+
+
+class TestEditAction(base.FunctionalTestBase):
+    async def test_pr_to_draft_edit(self):
```

Comment:
Can we have 2 tests, one for each state?

Reply:
You think we should create the pull request as a draft directly and then check if it's correctly marked as `ready for review` in another test ?

Reply:
You should have:
- test 1 that creates PR as a draft and converts it to ready to review
- test2 that creates PR and converts it to a draft